app/bot_test.go

```diff
@@ -158,6 +158,9 @@ func TestPatchBot(t *testing.T) {
 		patchedBot, err := th.App.PatchBot(createdBot.UserId, botPatch)
 		require.Nil(t, err)
 
+		// patchedBot should create a new .UpdateAt time
+		require.NotEqual(t, createdBot.UpdateAt, patchedBot.UpdateAt)
```

Comment:
Not sure this tests the new code. This tests the bot update, but the modifications are about the user.

Reply:
The ideal test would be that the send updated user event was called with the correct parameter. But not sure if that is feasible. Not going to block this on that.

Reply:
Non-blocking but It could be tested similar to how we're doing in `api4/user_test.go` - look for `assertExpectedWebsocketEvent`.  Though it will require some test helpers to work here in app package.